src/App.js

```diff
@@ -1,10 +1,81 @@
 import React from 'react';
+import theHandyBag from './assets/bag_1.png'
+import theStylishBag from './assets/bag_2.png'
+import theSimpleBag from './assets/bag_3.png'
+import theTrendyBag from './assets/bag_4.png'
+import brand from './assets/brand.png'
+import ourStory from './assets/our_story.png'
+import Button from "./components/Button";
+import Product from "./components/Product";
+import Tile from "./components/Tile";
 import './App.css';
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button
+                    buttonType="button"
+                    children="to the collection"
+                    isDisabled={false}
+                />
+                <Button
+                    buttonType="button"
+                    children="shop all bags"
+                    isDisabled={false}
+                />
+                <Button
+                    buttonType="button"
+                    children="pre-orders"
+                    isDisabled={true}
+                />
+            </nav>
+            <main>
+                <Product
+                    info="Best seller"
+                    image={theHandyBag}
+                    description="The handy bag"
+                    price="€400,-"
```

Comment:
Je zou de elementen € en ,- kunnen verwerken in het component. Dat scheelt typen